dts/arm/atmel/samd.dtsi

```diff
@@ -25,6 +25,14 @@
 		reg = <0x20000000 0x8000>;
 	};
 
+	id: device_id@0 {
+		compatible = "atmel,sam0-id";
+		reg =	<0x0080A00C 0x4>,
+			<0x0080A040 0x4>,
+			<0x0080A044 0x4>,
+			<0x0080A048 0x4>;
+	};
+
```

Comment:
You wrote that SAMD2x has different addresses that SAMD5x.
Don't you need specific dts entries then?
I assume that samd is for both.

Reply:
No `samd.dtsi` is only for samd2x.
I would like to rename it, but maybe later when there aren't so many pull requests open anymore that will touch the file.

Reply:
Which one is for SAMD5x then?

Reply:
Currently Zephyr only supports SAMD20 and SAMD21 from the sam0 family.

Support for SAMD5x will come with #14685